numpy/lib/tests/test_function_base.py

```diff
@@ -1740,13 +1740,18 @@ def test_large_integers_increasing(self):
         # gh-11022
         x = 2**54  # loses precision in a float
         assert_equal(np.digitize(x, [x - 1, x + 1]), 1)
+        assert_equal(np.digitize(x, [x - 1, x + 1], False, True), 1)
+        with assert_raises(RuntimeError):
+            np.digitize(x, [x - 1, 2 ** 63 - 1], False, True)
```

Comment:
The following should not fail, but does:
```python
np.digitize(np.uint64(x), np.array([x - 1, 2 ** 63 - 1], dtype=np.uint64), False, True)
```

Reply:
I agree that it would be nice to support that but I tried implementing a branching logic for `uint64` but numpy behaved very funny with the large numbers (see above).